hexa/pipelines/schema/mutations.py

```diff
@@ -330,6 +332,12 @@ def resolve_upload_pipeline(_, info, **kwargs):
         return {"success": False, "errors": ["INVALID_TIMEOUT_VALUE"]}
     except PermissionDenied:
         return {"success": False, "errors": ["PERMISSION_DENIED"]}
+    except IntegrityError as e:
+        if isinstance(
+            e.__cause__, UniqueViolation
+        ) and UNIQUE_PIPELINE_VERSION_NAME in str(e):
+            return {"success": False, "errors": ["DUPLICATE_PIPELINE_VERSION_NAME"]}
+        raise
```

Comment:
I am not catching the `IntegrityError` because I think that it's unexpected and should raise the full exception